modules/moderation.py

```diff
@@ -268,350 +331,203 @@ async def _unmuting(self, ctx, member: discord.Member, *, reason='-No reason spe
+        userDB = mclient.bowser.users
+        loop = self.bot.loop
+        punCount = db.count_documents({'active': True, 'type': {'$in': ['tier1', 'tier2', 'tier3']}})
+        if not punCount > 0:
+            return await ctx.send('nothing to do!')
+
+        failures = 0
+        for doc in db.find({'active': True, 'type': {'$in': ['tier1', 'tier2', 'tier3']}}):
+            strikeCount = int(doc['type'][-1:]) * 4
+            try:
+                member = await ctx.guild.fetch_member(doc['user'])
 
-                embed = discord.Embed(color=0xD0021B, timestamp=datetime.datetime.utcnow())
-                embed.set_author(name=f'Warning made permanent | {member} ({member.id})')
-                embed.set_footer(text=warnPun['_id'])
-                embed.add_field(name='User', value=member.mention, inline=True)
-                embed.add_field(name='Reason', value='Moderator decision to make warning permanent')
-                await self.modLogs.send(embed=embed)
-                await resp.delete()
-                return await ctx.send(f'{config.greenTick} Warning review complete for {str(member)} ({member.id}). Will be made permanent; no further warning review reminders will be sent.')
+            except discord.NotFound:
+                userDB.update_one({'_id': doc['user']}, {'$set': {'migrate_unnotified': True}}) # Set flag for on_member_join to instruct of new system should they return
+                continue # TODO: handle this in core
+
+            db.update_one({'_id': doc['_id']}, {'$set': {'active': False}})
+            docID = await tools.issue_pun(doc['user'], self.bot.user.id, 'strike', f'[Migrated] {doc["reason"]}', strike_count=strikeCount, context='strike-migration', public=False)
+            self.taskHandles.append(loop.call_later(5, asyncio.create_task, self.expire_actions(docID, ctx.guild.id)))
+            userDB.update_one({'_id': member.id}, {'$set': {'strike_check': time.time() + (60 * 60 * 24 * 7)}}) # Setting the next expiry check time
+
+            explanation = """Hello there **{}**,\nI am letting you know of a change in status for your active level {} warning issued on {}.\n\nThe **/r/NintendoSwitch** Discord server is moving to a strike-based system for infractions. Here is what you need to know:\n\* Your warning level will be converted to **{}** strikes.\n\* __Your strikes will decay at the same rate as warnings previously did__. Each warning tier is the same as four strikes with one strike decaying per-week instead of one warn level per four weeks.\n\* You will no longer have any permission restrictions you previously had with this warning. Moderators will instead restrict features as needed to enforce the rules on a case-by-case basis.\n\nStrikes will allow the moderation team to weigh rule-breaking behavior better and serve as a reminder to users who may need to review our rules. Please feel free to send a modmail to @Parakarry (<@{}>) if you have any questions or concerns.""".format(
```

Comment:
It seems weird to explain that "Your strikes will decay at the same rate as warnings previously did," if the units are fundamentally different, perhaps  "decay at *a equivalent* rate"

Reply:
> Each warning tier is the same as four strikes with one strike decaying per-week instead of one warn level per four weeks.

Prehaps change to

> Each warning tier is *equivalent* to four strikes*, where* one strike *decays* per week instead of one warn level per four weeks

Reply:
Maybe include that !history can now we run by users in this explanation

Reply:
First part resolved in `d387c7e`, !history explanation still need to be done.